safe_cli/operators/hw_accounts/hw_account.py

```diff
@@ -0,0 +1,67 @@
+import re
+from abc import ABC, abstractmethod
+
+from eth_typing import ChecksumAddress
+
+BIP32_ETH_PATTERN = r"^(m/)?44'/60'/[0-9]+'/[0-9]+/[0-9]+$"
+BIP32_LEGACY_LEDGER_PATTERN = r"^(m/)?44'/60'/[0-9]+'/[0-9]+$"
+
+
+class InvalidDerivationPath(Exception):
+    message = "The provided derivation path is not valid"
+
+
+class HwAccount(ABC):
+    def __init__(self, derivation_path: str, address: ChecksumAddress):
+        self.derivation_path = derivation_path
+        self.address = address
```

Comment:
Why is address passed to this method, instead of being calculated from derivation path on the __init__ method?